### PR TITLE
Bluetooth: btp: Extend BTP with new MESH commands

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This adds new BTP Mesh commands, events and it's implementations.

### CODE REVIEW

File: include/bluetooth/testing.h [hunk 1]
```diff
@@ -60,6 +65,70 @@ void bt_test_cb_unregister(struct bt_test_cb *cb);
...
+ *
+ *  Called upon successful model bind.
+ *  This will call registered mesh_model_bound callbacks.
+ *
+ *  @param addr Remote address
+ *  @param model Model
+ *  @param key_idx AppKey index
+ */
+void bt_test_mesh_model_bound(u16_t addr, struct bt_mesh_model *model,
+			      u16_t key_idx);
```
Comment (jhedberg) [inline include/bluetooth/testing.h]: Isn't this an internal API? Only public APIs, i.e. those that apps call should reside under include/bluetooth/. It seems to me you're only calling this API from within the mesh stack.

Comment (MariuszSkamra) [reply include/bluetooth/testing.h]: OK, I've move this to internal API

**Comment by MariuszSkamra** [comment]
- Moved function declarations to internal API

**Comment by MariuszSkamra** [comment]
- Added another BTP event ti indicate expiration of segmented message incomplete timer

**Review by sjanc — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**